File: Level 2/마법의 엘리베이터/solution.py
def solution(storey):
    count = 0

    while storey:
        rest = storey % 10
        storey //= 10

        if rest < 5 or (rest == 5 and storey % 10 < 5):
            count += rest
        else:
            count += 10 - rest
            storey += 1

    return count

File: Level 2/마법의 엘리베이터/solution.js
function solution(storey) {
  let count = 0;

  while (storey) {
    const rest = storey % 10;
    storey = Math.floor(storey / 10);

    if (rest < 5 || (rest === 5 && storey % 10 < 5)) {
      count += rest;
    } else {
      count += 10 - rest;
      storey++;
    }
  }

  return count;
}
